pageserver/src/page_service.rs

```diff
@@ -550,8 +493,24 @@ impl PageServerHandler {
         // Import basebackup provided via CopyData
         info!("importing basebackup");
         pgb.write_message(&BeMessage::CopyInResponse)?;
-        let reader = CopyInReader::new(pgb);
-        import_basebackup_from_tar(&*timeline, reader, base_lsn)?;
+        pgb.flush().await?;
+
+        // import_basebackup_from_tar() is not async, mainly because the Tar crate
```

Comment:
I remember we used [async_compression](https://docs.rs/async-compression/latest/async_compression/) crate to pack remote storage files together, probably we can use it there too. Can be a follow up

@SomeoneToIgnore What do you think about async_compression?

Reply:
It was slow to compile and did not include the container library (i.e. working with tar per se), so does not look like a good choice for this task.